solution/backend/regulations/admin.py

```diff
@@ -1,10 +1,99 @@
-from django.contrib import admin
+import re
 
+from django.contrib import admin, messages
```

Comment:
Just addin the note to reorganize the imports.  I know you said you would.

Reply:
Resolved in 2ea4ae6